libdnf5/module/module_sack_impl.hpp

```diff
@@ -181,6 +184,7 @@ class ModuleSack::Impl {
     std::unique_ptr<libdnf5::solv::SolvMap> excludes;
     std::map<Id, ModuleItem *> active_modules;
     std::vector<libdnf5::solv::IdQueue> modules_to_enable;
+    std::vector<std::vector<std::tuple<ProblemRules, Id, Id, Id, std::string>>> solver_problems;
```

Comment:
I am not sure about this. It would be better to return the problems from `ModuleSack::Impl::module_solve` and then from `ModuleSack::resolve_active_module_items`, but it's on the public API. Hence this workaround. But maybe we could change the return type if we are doing more changes that break the API.

Reply:
I am so sorry, but somehow I missed some connection in the comment.